apps/modernization-ui/src/apps/page-builder/components/QuickConditionLookup/QuickConditionLookup.tsx

```diff
@@ -41,6 +41,7 @@ export const QuickConditionLookup = ({ modal, addConditions }: Props) => {
     const { state } = useContext(UserContext);
     const { currentPage, setCurrentPage, pageSize, setPageSize } = useContext(ConditionsContext);
 
+    console.log('test', conditions);
```

Comment:
Issue: `console.log`

Reply:
fixed

Reply:
Ok i think i have it fixed with adding location to the useEffect, now we update the page number and page size whenever the query params change.